core/src/components/item/item.scss

```diff
@@ -412,7 +412,7 @@ button, a {
   box-sizing: border-box;
 }
 
-:host(.item-label-stacked),
```

Comment:
Why are we only removing the `stacked` style here? Other parts of the PR removed both.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/29102/commits/1e480e7e3c45629b89587142766d926051a00d32